From e1d591e5880a3f268c477cf1458b68217990eac9 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Tue, 16 Jul 2019 19:04:49 +0000
Subject: [PATCH] joyent/bugview#36 Update to jiramark@0.5.0 Reviewed by:
 Joshua M. Clulow <jmc@joyent.com> Approved by: Joshua M. Clulow
 <jmc@joyent.com>

---
 Makefile               | 2 +-
 package.json           | 2 +-
 templates/primary.html | 3 +--
 3 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 9a08594..0283d07 100644
--- a/Makefile
+++ b/Makefile
@@ -44,7 +44,7 @@ endif
 #
 
 .PHONY: all
-all: 0-npm-stamp | $(NPM_EXEC)
+all: | $(NPM_EXEC) 0-npm-stamp
 
 0-npm-stamp:
 	$(NPM) install
diff --git a/package.json b/package.json
index 5102fea..5cb6862 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
     "assert-plus": "^1.0.0",
     "bunyan": "~1.8.12",
     "ent": "~2.0.0",
-    "jiramark": "0.4.0",
+    "jiramark": "0.5.0",
     "restify": "~4.3.4",
     "vasync": "^2.2.0",
     "verror": "^1.10.0"
diff --git a/templates/primary.html b/templates/primary.html
index 0119e93..3591a0b 100644
--- a/templates/primary.html
+++ b/templates/primary.html
@@ -37,8 +37,7 @@
     /* Style processed JIRA markup. */
     div.panel {
       border: 2px solid black;
-      margin-left: 1em;
-      margin-right: 1em;
+      margin: 1em;
     }
     div.panelContent {
       max-height: 75em;
-- 
2.21.0

